api.py

```diff
@@ -52,10 +52,10 @@ def get_geocoords(address):
 		# and its corresponding geocoords in its own table in the event that, say, 
 		# there are many different users living in the same apartment building. 
 		# This would avoid the repetition of querying geocoords for the same street address each time.
-		return (37.6,-95.665)
```

Comment:
When you want to return a tuple you can just past params separated by coma and that will return a tuple.